Makefile

```diff
@@ -65,22 +38,22 @@ clean:
 
 all: dev_env format build_debug clippy test doc
 
-format: $(FMT_LOC) $(TOMLFMT_LOC)
+format: $(FMT) $(TOMLFMT)
 	@$(CARGO) fmt
 	@$(CARGO) tomlfmt -p Cargo.toml
 
-build_debug: build_env
+build_debug:
 	@$(CARGO) build --all --all-targets
 
-clippy: $(CLIPPY_LOC)
+clippy: $(CLIPPY)
 	@$(CARGO) clippy --all-targets -- -D warnings
 
-test: build_env
+test:
 	@$(CARGO) test --all
 
-doc: build_env
+doc:
 	@$(CARGO) doc
 
-check_format: $(FMT_LOC) $(TOMLFMT_LOC)
+check_format: $(FMT) $(TOMLFMT)
 	@$(CARGO) fmt -- --check
```

Comment:
Ideally, it should be like this no?
```suggestion
	@$(FMT) -- --check
```

Reply:
Well, not if we want to use `rustup run`... Ok, I can try better.

Reply:
It is a bit weird that we use `$(CARGO) fmt -- --check` on the existence of `cargo-fmt` in the path. We could use the `rustup run` command to also check for the presence of `rustfmt`:

![image](https://user-images.githubusercontent.com/5486389/66981656-19ffcf80-f100-11e9-9144-794f39b58fe0.png)

Reply:
ok, I know what I want to do :)